content/develop/data-types/probabilistic/configuration.md

```diff
@@ -0,0 +1,166 @@
+{{< /note >}}
+
+
+## Redis probabilistic data structure configuration parameters
+
+The following table summarizes which Bloom filter configuration parameters can be set at run-time, and compatibility with Redis Software and Redis Cloud
+
+| Old parameter name | New parameter name                          | Run-time           | Redis<br />Software | Redis<br />Cloud |
+| :-------           | :-------                                    | :-------           | :-------              | :-------         |
+| ERROR_RATE         | [bf-error-rate](#bf-error-rate)             | :white_check_mark: | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Supported">&#x2705; Flexible & Annual</span><br /><span title="Not supported"><nobr>&#x274c; Free & Fixed</nobr></span> |
+|                    | [bf-expansion-factor](#bf-expansion-factor) | :white_check_mark: |||
+| INITIAL_SIZE       | [bf-initial-size](#bf-initial-size)         | :white_check_mark: | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Supported">&#x2705; Flexible & Annual</span><br /><span title="Not supported"><nobr>&#x274c; Free & Fixed</nobr></span> |
+
+The following table summarizes which Cuckoo filter configuration parameters can be set at run-time, and compatibility with Redis Software and Redis Cloud.
+
+| Old parameter name | New parameter name                          | Run-time           | Redis<br />Software | Redis<br />Cloud |
+| :-------           | :-------                                    | :-------           | :-------              | :-------         |
+|                    | [cf-bucket-size](#cf-bucket-size)           | :white_check_mark: |||
+|                    | [cf-initial-size](#cf-initial-size)         | :white_check_mark: |||
+|                    | [cf-expansion-factor](#cf-expansion-factor) | :white_check_mark: |||
+| CF_MAX_EXPANSIONS  | [cf-max-expansions](#cf-max-expansions)     | :white_check_mark: | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Supported">&#x2705; Flexible & Annual</span><br /><span title="Not supported"><nobr>&#x274c; Free & Fixed</nobr></span> |
+|                    | [cf-max-iterations](#cf-max-iterations)     | :white_check_mark: |||
+
+{{< warning >}}
+A filter should always be sized for the expected capacity and the desired error rate.
+Using the `INSERT` family commands with the default values should be used in cases where many small filters exist and the expectation is most will remain at around the default sizes.
+Not optimizing a filter for its intended use will result in degradation of performance and memory efficiency.
+{{< /warning >}}
+
+## Default parameters for Bloom filters
+
+### bf-error-rate
+
+Default error ratio for Bloom filters.
+
+Type: double
+
+Valid range: `[0 .. 1]`; consider limiting the range to `[0 .. 0.25]`.
```

Comment:
Why not a table?
Configuration parameters since version 8.0.0
Name, description, type, range, default value

Reply:
All config. pages need to have the same user experience. Time series has a few very long discussions, some with tables of their own. Markdown (at least, the version we're using) doesn't support tables within tables.